### PR TITLE
Return BotKube version with response to ping

### PR DESCRIPTION
**Pull request by PrasadG193**
Fixes: https://github.com/infracloudio/botkube/issues/62

### CODE REVIEW

**Comment by PrasadG193** [comment]
image

File: helm/botkube/values.yaml [hunk 1]
```diff
@@ -6,7 +6,6 @@ replicaCount: 1
 
 image:
   repository: infracloud/botkube
-  tag: "0.6"
```
Comment (ssudake21) [inline helm/botkube/values.yaml]: How we are deciding image version after removal of tag ?

File: pkg/execute/executor.go [hunk 2]
```diff
@@ -213,7 +222,16 @@ func runNotifierCommand(args []string, clusterName string, isAuthChannel bool) s
 	return printDefaultMsg()
 }
 
-func runPingCommand(args []string, clusterName string) string {
+func findBotKubeVersion() (versions string) {
+	args := []string{"-c", fmt.Sprintf("%s version --short=true | awk '/Server/ {print $3}'", kubectlBinary)}
+	cmd := exec.Command("sh", args...)
+	k8sVersion, _ := cmd.CombinedOutput()
```
Comment (ssudake21) [inline pkg/execute/executor.go]: please handle error here

Comment (ssudake21) [reply pkg/execute/executor.go]: LOG warning/error based on severity of error

File: pkg/execute/executor.go [hunk 3]
```diff
@@ -213,7 +222,16 @@ func runNotifierCommand(args []string, clusterName string, isAuthChannel bool) s
...
+
+	botkubeVersion := os.Getenv("BOTKUBE_VERSION")
```
Comment (ssudake21) [inline pkg/execute/executor.go]: If BotKube version is empty, we should show Unknown or nil

File: helm/botkube/values.yaml (hunk 1 above)
Comment (PrasadG193) [reply helm/botkube/values.yaml]: We'll use chart version as an image tag
```
          image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
```

**Pull request merged**